### ISSUE TITLE
RedisBungee support

### ISSUE THREAD

**Issue by Drc-DEV** [NuVotifier/NuVotifier#157]
Can you add support for RedisBungee in order to use the pluginMessaging system instead of the proxy? RedisBungee's API should have all you need in order to do this. https://github.com/minecrafter/RedisBungee/wiki/API

**Comment by Ichbinjoe** [comment]
Can you describe your setup which requires that you use RedisBungee over the current plugin messaging and proxy forwarding?

**Comment by Drc-DEV** [comment]
A player that votes in a different bungee server than the one whose IP is specified in the votifier settings of the voting website is not registered if I have pluginMessaging. And proxy mode is not really suitable for my environment. I have 5 bungee instances, I can only set 1 ip on the vote website.

**Comment by Ichbinjoe** [comment]
I'm not sure why proxy mode isn't suitable for your environment. It was basically designed for multi-bungee setups.

For more context, most of the time you can enter a DNS name into websites instead of just an IP - this is required functionality for some shared hosts and home hosters using dynamic IP. In your case you can set up round robin to cycle through your online proxies. 

Am I missing some functionality that you are missing? It sounds like proxy mode fits your use case. I don't mean to be difficult but unlike basically all other plugins I want to avoid making changes / adding unnecessary features due to risk of stability.

**Comment by Ichbinjoe** [comment]
I think this has actually been suggested before. Someone made a RedisBungee fork of this plugin but the code quality, backwards compatibility and configurability weren't good enough for me to merge it in.

**Comment by Drc-DEV** [comment]
I'll try and see if the proxy thing works. But all the servers are in a pterodactyl panel which means each one is inside a docker container and I don't know how it will perform. I'd really like to use the pluginMessaging instead. I'll update the status as soon as I get feedback from players on if it works and if so if it is stable/reliable.

**Comment by Drc-DEV** [comment]
Anything works fine on the Bungees, they get the vote from the websites but then it is " Unable to send vote to /IP:PORT". Yeah I made sure the port was open, it was allocated in the docker container in which the destination server is in, I made sure to put 0.0.0.0 on the spigot server as host and configure the same port as in the server proxy config on bungee, alongside the same token.
I have no strange firewall setup, just a default iptables. I've seen countless people struggling to setup votifier on their pterodactyl panel, and I don't blame this on Votifier, but I think the easiest way to solve the issue would be just to make the pluginMessaging work also for Redisbungee and ditch the proxy setup altogether.
